tracing/tracer.go

```diff
@@ -82,6 +83,14 @@ type Tracer struct {
 // If it fails to do so, UndefinedIP will be used instead,
 // and the error will be logged if logger is non-nil.
 func InitGlobalTracer(cfg Config) error {
+	logger := cfg.Logger
+	if logger == nil {
+		logger = log.NopWrapper
+	}
+	if internalv2compat.V2TracingEnabled() {
+		logger(context.Background(), `v2 tracing is enabled; skipping v0 tracer configuration`)
```

Comment:
nit:
```suggestion
	if internalv2compat.V2TracingEnabled() {
		cfg.Logger.Log(context.Background(), `v2 tracing is enabled; skipping v0 tracer configuration`)
```
see https://pkg.go.dev/github.com/reddit/baseplate.go/log#Wrapper.Log